include/networkit/graph/GraphBuilder.hpp

```diff
@@ -76,8 +87,9 @@ class GraphBuilder {
      * @param n Number of nodes.
      * @param weighted If set to <code>true</code>, the graph has edge weights.
      * @param directed If set to @c true, the graph will be directed.
+     * @param parallel If set to @c true, the graph will be build in parallel.
      */
-    GraphBuilder(count n = 0, bool weighted = false, bool directed = false);
+    GraphBuilder(count n = 0, bool weighted = false, bool directed = false, bool parallel = false);
```

Comment:
Given the performance is increasing (as well if executed on one thread) and If the result is the same (and it should be maybe except for the order of nodes in the adjacency list) for both variants, then `parallel` can be set to true.